contracts/svm/programs/express_relay/README.md

```diff
@@ -19,28 +19,34 @@ To integrate with Express Relay, the integrating program needs to make the follo
 
 ## Example Integration
 
-Integrating programs can use the `check_permission` helper method defined in the Express Relay SDK:
+Integrating programs can use the `check_permission_cpi` helper method defined in the Express Relay SDK:
```

Comment:
I think we can add some snippet on how config_router and express_relay_metadata look like:

```rust
#[account(seeds = [SEED_CONFIG_ROUTER, router.key().as_ref()], bump, seeds::program = express_relay.key())]
pub config_router: UncheckedAccount<'info>,
#[account(seeds = [SEED_METADATA], bump, seeds::program = express_relay.key())]
pub express_relay_metadata: Account<'info, ExpressRelayMetadata>,
```